docs/app/content-collections.md

```diff
@@ -1,168 +1,113 @@
-#2 above.
+1. Navigate to the Content Module and select the desired Collection.
+2. Select <span mi icon>search</span> in the Page Header and a search bar will appear.
+3. Type in the string or number to search by. The page will display Items that have a matching string or number value in
+   one or more of its Fields.
 
-:::
+## Filter Items
 
-## Manually Sorting Items
+![Filter Items](image.webp)
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to sort
-3. Choose a layout that supports manual sorting (eg: the default "table" mode)
-4. Enable manual sorting. (eg: for the table layout, click on the Sort column's header)
-5. Drag items by their handle into a new position
+It is possibly to conditionally [filter](/app/filters/) which Items are displayed on the Collection page. To filter a
+Collection's Items, follow the steps below.
 
-The new sort order is saved automatically, so make sure you wait until the App's loading indicator finishes before
-navigating away from the page.
+1. Navigate to the Content Module and select the desired Collection.
+2. Select <span mi icon>filter_list</span> in the Page Header.
+3. Apply any [Filters](/app/filters/) as needed.
 
-::: warning Requires Configuration
+## Manually Sort Items
 
-This feature is only available if a Sort column has been configured in the Collection's data model settings.
-
-:::
+<video autoplay muted loop controls title="Batch Edit Items">
+	<source src="" type="video/mp4" />
+</video>
 
-## Ordering Items
+Drag-and-drop Items to custom-sort them as desired. This is only possible on certain Layouts, such as the default
+[Table Layout](/app/content-layouts/#table-layout). To sort a Collection's Items, follow the steps below.
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to a collection of items
-3. Choose a layout that supports manual ordering (eg: the default "table" mode)
-4. Set the Order Field and Order Direction
-   - Table Layout: Click on the field's column header, clicking again changes order direction
-   - Card Layout: Adjust the two dropdowns in the layout's subheader
+1. Navigate to the Content Module and select the desired Collection.
+2. In the Sidebar, choose a [Layout](/app/content-layouts) that supports manual sorting.
+3. Enable manual sorting:
+   - For the Table Layout, toggle <span mi icon>sort</span> in the configured Sort column.
+4. Drag Items by their handle into a new position.
 
-::: tip Sort vs Order
+::: warning Sorting Requires Configuration
 
-**Ordering** is a non-destructive action (does _not_ change your data) used to browse items alphabetically or
-numerically. **Sorting** (when enabled) saves a value in a [Sort](/reference/query/#sort) field, which can then be used
-to return data in a custom order.
+Only available if a Sort column has been configured in the Collection's Data Model Settings.
```

Comment:
This needs to link to config, but config isn't ready yet. I'm searching for a good way to save these spots so I can quickly come back later. lmk if you have any ideas.

Reply:
html comment with @todo in the actual docs

Reply:
```html
<!-- @TODO add link to config reference once ready -->
```